packages/amplify-cli/package.json

```diff
@@ -76,7 +76,6 @@
     "env-editor": "^0.5.0",
     "esm": "^3.2.25",
     "execa": "^4.1.0",
-    "figlet": "^1.2.4",
```

Comment:
This is in yarn.lock as well. Do we normally update that file at the same time?

Reply:
Yes, we should have the updated `yarn.lock` included here

Reply:
Added now.